Makefile

```diff
@@ -271,6 +280,12 @@ failpoint-disable: install-go-tools
 	# Restoring failpoints...
 	@$(FAILPOINT_DISABLE)
 
+split:
+	go list ./... | grep -v -E  "github.com/tikv/pd/server/api|github.com/tikv/pd/tests/client|github.com/tikv/pd/tests/server/tso" > packages.list;\
```

Comment:
What about these three packages?

Reply:
iit will remove server/api、/tests and tso package after daily ci integrate all verify ci.